mojio_sdk/engine_oil.py

```diff
@@ -0,0 +1,15 @@
+class EngineOil:
```

Comment:
I don't see any of the attributes of this class in my JSON response. I guess the capabilities of the Mojio device depends upon which data it can retrieve via the ADB connector. My car is a 2018 Opel Insignia. See how the response for my car looks like in https://github.com/simontb/mojio-home-assistant/issues/7#issuecomment-1128092812

I saw that in fuel.py you checked for the presence of attributes. Maybe that should be done for all advanced attributes, because we don't know which attributes are available for whom.

Reply:
Yeah, I'm trying to be careful and null-check everything.  Properties will depend on the SyncUp device, car, and probably country.  For example, mine shows oil temp, but it never gets updated.

Reply:
Completed.